### DISCUSSION TITLE
香港傳統漢字 or 臺灣正體

### DISCUSSION THREAD

**Discussion by gyrann** [rime/rime-cantonese#169] [Q&A] [answered]
Hi, I wonder when I install the 粵拼 keyboard, has it been automatically set to 香港傳統漢字 or is it 臺灣正體? 
I am not a native speaker, so it is hard for me to distinguish between them. I'd be very grateful if someone could please tell me how to set the 粵拼 keyboard in 香港傳統漢字.

**Comment by laubonghaudoi** [comment, marked as answer]
默認就係港標嘅。港標同台標嘅區別睇呢兩個文件
https://github.com/BYVoid/OpenCC/blob/master/data/dictionary/HKVariants.txt
https://github.com/BYVoid/OpenCC/blob/master/data/dictionary/TWVariants.txt
